bin/knuckle_cluster

```diff
@@ -31,7 +31,7 @@ rescue => e
 end
 
 kc = KnuckleCluster.new(
-  config
+  **config
```

Comment:
`config` is a `Hash` but `KnuckleCluster.new` expects keyword arguments. This was supported until Ruby 2.6 but is not recommended in 2.7. [This behavior will change in Ruby 3.0.](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/)

`**` here converts the `Hash` to keyword arguments, fixing the deprecation warnings.

https://github.com/envato/knuckle_cluster/blob/6ad79c88149f113c2668353cebd2411b9acf82ed/bin/knuckle_cluster#L27